src/pages/Explore/Explore.tsx

```diff
@@ -74,60 +115,145 @@ export const Explore = memo(() => {
-          onClick={handlePoolsClick}
-        />
-        <ExploreCard
-          title='explore.lending.title'
-          body='explore.lending.body'
-          icon={lendingIcon}
-          bg='linear-gradient(161deg, #D69E2E 6.22%, #935F22 87.07%)'
-          onClick={handleLendingClick}
-        />
-        <ExploreCard
-          title='explore.tcy.title'
-          body='explore.tcy.body'
-          icon={tcyIcon}
-          bg='linear-gradient(159deg, #319795 2.01%, #215063 86%);'
-          onClick={handleTCYClick}
-        />
-        <ExploreCard
-          title='navBar.defi'
-          body='defi.myPositionsBody'
-          icon={defiIcon}
-          bg='linear-gradient(161deg, #D6BCFA 6.22%, #553C9A 87.07%)'
-          onClick={handleEarnClick}
-        />
+
+        <Box
+          display='flex'
+          flexDir='column'
+          flex='1 1 auto'
+          height={
+            isSearching
+              ? 'calc(100vh - var(--mobile-nav-offset) - env(safe-area-inset-bottom) - var(--safe-area-inset-bottom) - 98px - 1rem)'
+              : 'auto'
+          }
+          mb={2}
+        >
+          <Box as='form' flex='0 0 auto' mb={3} visibility='visible' onSubmit={handleSubmit}>
+            <InputGroup size='md'>
+              <InputLeftElement pointerEvents='none' zIndex={1}>
+                <SearchIcon color='text.subtle' fontSize='md' />
+              </InputLeftElement>
+              <Input {...inputProps} />
+              <InputRightElement>
+                {isSearching ? <IoClose onClick={handleClearSearch} /> : null}
+              </InputRightElement>
+            </InputGroup>
+          </Box>
+          {isSearching ? (
+            <AssetList
+              assets={assetResults}
+              handleClick={handleAssetClick}
+              disableUnsupported={false}
+              rowComponent={AssetSearchRow}
+            />
+          ) : null}
+        </Box>
+
+        <Box display={isSearching ? 'none' : 'block'}>
+          <Flex flexDir='column' gap={6}>
+            <Carousel autoPlay showDots options={carouselOptions}>
+              <ExploreCard
+                title='navBar.foxEcosystem'
+                body='explore.foxEcosystem.body'
+                icon={foxIcon}
+                onClick={handleFoxClick}
+              />
+              <ExploreCard
+                title='explore.pools.title'
+                body='explore.pools.body'
+                icon={poolsIcon}
+                onClick={handlePoolsClick}
+              />
+              <ExploreCard
+                title='explore.tcy.title'
+                body='explore.tcy.body'
+                icon={tcyIcon}
+                onClick={handleTCYClick}
+              />
+              <ExploreCard
+                title='navBar.defi'
+                body='defi.myPositionsBody'
+                icon={defiIcon}
+                onClick={handleEarnClick}
+              />
```

Comment:
@NeOMakinG we keeping this in the end?

Reply:
https://github.com/shapeshift/web/pull/10191#discussion_r2256549242

I responded here, wdyt?

Reply:
Closing for the sake of progression, we can always remove it at any time